apps/system/js/bluetooth_transfer.js

```diff
@@ -530,6 +531,38 @@ var BluetoothTransfer = {
       detail: details}));
   },
 
+  /**
+   * Handle related system notification message operation after reboot.
+   * Need close notification message by ourself.
+   *
+   * @param {Object} message system notification message
+   */
+  handleSystemMessageNotification: function(message) {
+    this.debug('Received system message: ' + JSON.stringify(message));
+    var evt = {
+      fileName: message.body,
+      contentType: message.data.contentType
+    };
+    this.openReceivedFile(evt);
+    this.closeSystemMessageNotification(message);
+  },
+
+  /**
+   * Close the notification message.
+   *
+   * @param {Object} message system notification message
+   */
+  closeSystemMessageNotification: function(msg) {
```

Comment:
reference https://github.com/mozilla-b2g/gaia/blob/master/apps/system/js/findmydevice_launcher.js#L43
removed the origin tag part because it is not triggered.